code/modules/karma/karma.dm

```diff
@@ -117,6 +120,20 @@ var/list/karma_spenders = list()
 
 	spend_karma(pickedmob)
 
+/mob/verb/toggle_karma()
+	set name = "Toggle Karma Gains"
```

Comment:
Remove one of the tabs from this line, and from the following two lines.

Reply:
Huh strange, could've sworn it was working earlier

Reply:
durrrrrr! fixed it and it compiles now thanks for the reviews